### PR TITLE
Prefix more WP versions with "WP-"

### PR DESCRIPTION
**Pull request by nylen**
Following up on #26, #57, and #59, this PR adds a `WP-` prefix to some more instances of WP version numbers in the code.

### CODE REVIEW

File: src/wp-content/themes/twentysixteen/inc/back-compat.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
  *
  * Prevents Twenty Sixteen from running on WordPress versions prior to 4.4,
```
Comment (senlin) [inline src/wp-content/themes/twentysixteen/inc/back-compat.php]: I think at the end of this line we need an additional `WP-`?

File: src/wp-includes/class-wp-editor.php [hunk 2]
```diff
@@ -57,7 +57,7 @@ private function __construct() {}
 	 *     @type string     $editor_class      Extra classes to add to the editor textarea element. Default empty.
 	 *     @type bool       $teeny             Whether to output the minimal editor config. Examples include
 	 *                                         Press This and the Comment editor. Default false.
-	 *     @type bool       $dfw               Deprecated in 4.1. Since 4.3 used only to enqueue wp-fullscreen-stub.js
+	 *     @type bool       $dfw               Deprecated in WP-4.1. Since 4.3 used only to enqueue wp-fullscreen-stub.js
```
Comment (senlin) [inline src/wp-includes/class-wp-editor.php]: The 4.3 also needs the `WP-` prefix I think?

File: src/wp-content/themes/twentysixteen/inc/back-compat.php (hunk 1 above)
Comment (nylen) [reply src/wp-content/themes/twentysixteen/inc/back-compat.php]: I didn't since it already says WordPress here, but I'm fine with being consistent too.

Comment (nylen) [reply src/wp-content/themes/twentysixteen/inc/back-compat.php]: Actually looking at the whole PR I think I did?

Comment (senlin) [reply src/wp-content/themes/twentysixteen/inc/back-compat.php]: > Actually looking at the whole PR I think I did?

Not sure I understand?

**Comment by nylen** [comment]
From my phone it looks like I've already made the changes you were thinking of. You may have been looking at only some of the commits when reviewing, instead of the final result. I'll take a closer look when I'm back at the computer later.

And yeah, GitHub "approval" generally means "this PR is ready to merge". Better not to approve until that's actually the case then.

File: src/wp-content/themes/twentysixteen/inc/back-compat.php (hunk 1 above)
Comment (senlin) [reply src/wp-content/themes/twentysixteen/inc/back-compat.php]: With this sentence you're actually right, "WordPress" make it clear already

**Comment by senlin** [comment]
When I look `src/wp-includes/class-wp-editor.php` L60 still says:

Deprecated in WP-4.1. Since 4.3 used only to enqueue wp-fullscreen-stub.js

Whereas I think it should read (emphasis mine):

Deprecated in WP-4.1. Since **WP-**4.3 used only to enqueue wp-fullscreen-stub.js

**Comment by nylen** [comment]
No worries. Thanks for reviewing.

**Pull request merged**